### ISSUE TITLE
Medial Mods

### ISSUE THREAD

**Issue by nschneid** [nert-nlp/cgel#49]
- For the auxes followed by an adverb, we should now have an extra VP layer per the Lexical Projection Principle, right? [VP is [Mod:AdvP not]] becomes [VP [VP is] [Mod:AdvP not]]

- Do we follow the same analysis or treat the complement as postposed for:
   * a moment to think **again** about the country that we want to be
   * the arrival **recently** of the new year
   * citizens have **for decades** taken it as a given that...

**Comment by BrettRey** [comment]
It's a very nice principle, and it would be lovely to be able to apply it everywhere, but it seems perverse to say that VPs with an aux have a useless VP hanging down below the modifier. It also implies that you could have _*he can do it not_. So I think this should be an exception.

As for the others, I think they're all postposed complements.

**Comment by nschneid** [comment]
If we narrow the LPP in #50, also look at Prenucleus function: we could have a lexical (non-phrasal) constituent there, such as V_aux or WH-word.

**Comment by nschneid** [comment]
In #50 we are changing the layered analysis of Noms, so that solves "arrival recently".

John's comments on the others:

> I had no input to any questions concerning VP-internal tree structures, and can't recall being asked by Rodney to comment on a draft of the relevant chapter. So I can't answer with much confidence to your questions about this! I do recall Rodney telling me however that the internal structure of VP would be different to what we decided for NP, in particular allowing ternary branching when there were two or more complements (inter alia, relative ordering seems stricter with verbal dependents than nominal ones).  CGEL seems to have it that both core (essentially NP) and non-core (e.g. PP) complements are daughters of VP and sisters of V. I may be wrong, but presumably the intention was that in an example like _think **again** about the country that we want to be_ there would be postposition of the heavy PP. The lowest VP would then contain the V think and a gap. Not at all sure about the structure beyond that, possibly another VP above this lower VP containing the adverb again, and then (?) yet a higher VP with the postposed PP.
>
> **However**, what worries me about a postposition analysis is that this order seems just as good with a very light PP:  _think **again** about it_. Maybe there's a case for thinking of just core complements (direct and indirect objects) as sisters of V, with both non-core complements and adjuncts attached to binary branching higher VPs and showing greater flexibility. However, that's not what CGEL seems to say. 
>
> With respect to your question (Is there a case in which a two-word VP would have a one-word VP as its head instead of a V?), I had assumed that the intention was that VPs were different to NPs. A two word VP consisting of just an intransitive verb and an adjunct would have this structure. But I may be wrong!
>
> Perhaps there are some trees somewhere in CGEL that I've missed with the answers, or maybe Geoff had closer contact with Rodney about these issues and has some insights?

> In _have **for decades** taken it as a given that..._ I would think for decades must be integrated, not a supplement. In this example the auxiliary have heads its own clause. It's not immediately obvious to me whether the adjunct is part of the higher clause or the lower clause (I will muse on that). But if it's part of the higher clause, it will raise exactly the same issues as in the previous example. 

"For decades" seems a clear case of postposing. Compare: _I have been playing chess for decades_ > _I have for decades been playing chess_.

"Think again about it", "think carefully about it", "think very very hard about it" (but "?think so hard that your head will explode about it"): I don't know if these tell us that even light PPs are heavy enough that they can be postposed, or that there is no "underlying" ordering preference between light(ish) adjuncts and light PPs.

What about single-word PPs? "He backed slowly away" and "I backed away slowly" both sound pretty acceptable to me, with only an information structure difference between the pairs. Same for "I drove (the car) slowly home" and "I drove (the car) home slowly". But *I drove slowly the car home.

*I paid again John $10 for the candy
*I paid John again $10 for the candy
I paid John $10 again for the candy

Is this grounds to say that a VP should allow a Mod to precede a PP complement on the same level (though intervening between the verb and direct or indirect object should be ungrammatical)? I suppose that would be less radical than saying a PP complement after an object should always be on a different level.

**Comment by BrettRey** [comment]
I don't think anyone's really happy with the postposing analysis. John writes,

>My preference for a change would be to only include core complements (direct and indirect objects) and probably phrasal verb particles as sisters of V, and to allow both non-core complements and adjuncts to be added recursively as sisters of VP, with labile ordering between them. In this analysis there would be a lower ternary-branching VP containing V paid, NP $10 and NP John, then a VP above that containing again, and a VP above that containing for the candy. No postposing. However, such an analysis would be a deviation from CGEL.

I'm comfortable with this deviation.

**Comment by nschneid** [comment]
But if there were no intervening Mod the complements would all be on the same level?

**Comment by BrettRey** [comment]
I'd say, looking at [10] on p. 219, only vi (complex transitive) and vii (ditransitive) would all be on the same level, regardless of any Mod. And I'd add to the any particles, so _Can you pick me up a pack?_ would be one VP.

**Comment by nschneid** [comment]
So then:

- [10ii] This [VP [V depends] [Comp:PP on the price]] (OK because it's binary)
- [10v] He [VP [VP [V blamed] [Obj:NP me]] [Comp:PP for the delay]] (PPs are non-core so on a separate level from core complements)
- [10vi] This [VP made [Obj:NP Ed] [PredComp:AdjP angry]]
- [10vii] She [VP gave [Obj_ind:NP him] [Obj_dir:NP some food]] (because both kinds of objects are core complements)
- [VP Pick [Obj_ind:NP me] [Particle: up] [Obj_dir:NP a pack]] (particles also core complements)
- [I [VP [VP [VP paid [Obj_ind:NP John] [Obj_dir:NP $10]] [Mod:AdvP again]] [Comp:PP for the candy]]]

Should ExtraposedSubj, ExtraposedObj, and DisplacedSubj also be considered core complements?

Would the Comp label ever be used for a *core* complement in CGELBank?

**Comment by BrettRey** [comment]
I think the bullets make sense. I think Comp will always be non-core, even with clausal complements (e.g., _I wonder again whether it might be like this._)

Displaced subjects must be core, I think.

The way we have Extraposition now is that it's a complement in the Clause, not the VP.

**Comment by nschneid** [comment]
"It isn't **that there aren't enough workers**, there just aren't enough..."—right now this is the one with ExtraposedSubj in VP. Is it really extraposition? You can't substitute the that-clause for *it*: *That there aren't enough workers isn't. Maybe "it" implicitly refers to something like "the problem".

(There are no ExtraposedObj instances in the corpus.)

**Comment by BrettRey** [comment]
No, I don't think it really is. I think this is the impersonal construction with _it_ as subject. But this is a case where Comp is a core complement,("We take the content clause in [17] to be an internal complement of the verb") so I guess I was, yet again, too hasty.

**Comment by nschneid** [comment]
An internal complement, yes—but a *core* internal complement?

Both of these seem OK and not obvious cases of postposing:
- It has seemed lately that he was trying to hide his true identity
- It turned out when I arrived that she had just been to the bank

**Comment by nschneid** [comment]
Oh and also: "it seemed to me that..." has "to me" as Comp. I assume it wouldn't be a core comp (otherwise we'd have to start classifying PPs as core vs. noncore, which would be a nightmare), and therefore the that-clause shouldn't be core either.

**Comment by BrettRey** [comment]
I'm starting to think that only objects are core. We get things like _it's probably the best choice_, _it seems largely a matter of historical accident_, and _that makes it automatically the winner_. This also means there's nothing special about post-auxiliary modifiers.

**Comment by nschneid** [comment]
Hmm...but *This made Ed again/yesterday angry.

**Comment by BrettRey** [comment]
Yes, that's terrible.

**Comment by nschneid** [comment]
Counts of different violations in the data:

	28 Mixing of core and non-core VP-internal functions (e.g. Obj and Comp): ['Head', 'Obj', 'Comp']
	 3 Mixing of core and non-core VP-internal functions (e.g. Obj and Comp): ['Head', 'Comp', 'Comp']
	 2 Mixing of core and non-core VP-internal functions (e.g. Obj and Comp): ['Head', 'Particle', 'Comp']
	 1 Mixing of core and non-core VP-internal functions (e.g. Obj and Comp): ['Head', 'Obj_ind', 'Obj_dir', 'Comp']
	 1 Mixing of core and non-core VP-internal functions (e.g. Obj and Comp): ['Head', 'Obj', 'Comp', 'Comp']

The last one is "take it as a given that...". Should it be ExtraposedSubj?

Additionally, if we move PredComp to non-core:

	 2 Mixing of core and non-core VP-internal functions (e.g. Obj and Comp): ['Head', 'Obj', 'PredComp']
	 1 Mixing of core and non-core VP-internal functions (e.g. Obj and Comp): ['Head', 'DisplacedSubj', 'PredComp']

**Comment by nschneid** [comment]
e.g.: they have [VP [Head:V sent] [Particle:PP over] [Obj:NP their top reporter Ahmed Mansour] [Comp:PP to the town]]
